app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/GoogleSignInWidget.kt

```diff
@@ -100,5 +109,6 @@ class GoogleSignInWidget : Fragment() {
 
     companion object {
         val RC_SIGN_IN: Int = "firebase auth".hashCode()
+        var isAdmin = false
```

Comment:
Shouldn't this be in the instance?

Reply:
I just found it easier to pass around in this fashion to represent the global state of the loginUI

Reply:
Why can it not be an instance variable on the Fragment? I don't understand your justification.

Reply:
Yes Please

Reply:
done, thank you